modules/ztp-configuring-hwevents-using-pgt.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-creating-hwevents_{context}"]
+= Configuring bare-metal events monitoring using PolicyGenTemplate CRs
+
+You can configure bare-metal hardware events for vRAN clusters that are deployed using the GitOps Zero Touch Provisioning (ZTP) pipeline.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Create a Git repository where you manage your custom site configuration data.
+
+[NOTE]
+====
+Multiple `HardwareEvent` resources are not permitted.
+====
+
+.Procedure
+
+. To configure the AMQ Interconnect Operator and the Bare Metal Event Relay Operator, add the following YAML to `spec.sourceFiles` in the `common-ranGen.yaml` file:
++
+[source,yaml]
+----
+# AMQ interconnect operator for fast events
+- fileName: AmqSubscriptionNS.yaml
+  policyName: "subscriptions-policy"
+- fileName: AmqSubscriptionOperGroup.yaml
+  policyName: "subscriptions-policy"
+- fileName: AmqSubscription.yaml
+  policyName: "subscriptions-policy"
+# Bare Metal Event Rely operator
+- fileName: BareMetalEventRelaySubscriptionNS.yaml
+  policyName: "subscriptions-policy"
+- fileName: BareMetalEventRelaySubscriptionOperGroup.yaml
+  policyName: "subscriptions-policy"
+- fileName: BareMetalEventRelaySubscription.yaml
+  policyName: "subscriptions-policy"
+----
+
+. Add the `Interconnect` CR to `.spec.sourceFiles` in the site configuration file, for example, the `example-sno-site.yaml` file:
++
+[source,yaml]
+----
+- fileName: AmqInstance.yaml
+  policyName: "config-policy"
+----
+
+. Add the `HardwareEvent` CR to `spec.sourceFiles` in your specific group configuration file, for example, in the `group-du-sno-ranGen.yaml` file:
++
+[source,yaml]
+----
+- fileName: HardwareEvent.yaml
+  policyName: "config-policy"
+  spec:
+    nodeSelector: {}
+    transportHost: "amqp://<amq_interconnect_name>.<amq_interconnect_namespace>.svc.cluster.local" <1>
+    logLevel: "info"
+----
+<1>  The `transportHost` URL is composed of the existing AMQ Interconnect CR `name` and `namespace`. For example, in `transportHost: "amqp://amq-router.amq-router.svc.cluster.local"`, the AMQ Interconnect `name` and `namespace` are both set to `amq-router`.
```

Comment:
For me, the `<1>` doesn't appear rendered in the doc preview. I think you need a `+` after line 62, but it might show in your local preview. Wanted to point out just in case!

Reply:
I think the preview is not updating properly, let me check what I can do...

Reply:
Preview updated and <1> renders correctly!